src/App.js

```diff
@@ -1,25 +1,26 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
```

Comment:
You should have extracted [`Switch`](https://reactrouter.com/web/api/Switch) as well.